docs/policy/comanage-instructions-user.md

```diff
@@ -0,0 +1,54 @@
+Registering for the OSG COManage
+================================
+
+OSG is using a new identity management system called COManage.
+This system will be used for managing access to OSG staff-internal monitoring webpages at UNL
+and managing access for OASIS logins.
+
+Follow the instructions below to register with COManage and join the OSG group.
+
+!!! note
+    This page is for users who want to register with COManage.
+    If you are a COManage Admin and want to approve a user registration,
+    go to the [Approving COManage Registrations](/policy/comanage-instructions-admin) page instead.
+
+
+1.  Go to <https://registry.cilogon.org/registry/co_petitions/start/coef:64>
+
+1.  Search for your institution and select it.
+
+1.  Log in with your institutional credentials.
+    You should reach the "OSG Registration: Self Signup With Approval" page.
+
+1.  Click "BEGIN".
```

Comment:
Personally, I prefer a style where you both label and name the UI control, e.g.: Click the “BEGIN” button. But do whatever is consistent with the rest of this documentation area.